taskcluster/fenix_taskgraph/parameters.py

```diff
@@ -14,13 +14,23 @@
 
 from .release_promotion import read_version_file
 
+
+def get_defaults(repo_root):
+    return {
+        "pull_request_number": None,
+        "release_type": "",
+        "next_version": "",
+        "version": "",
+    }
+
+
 extend_parameters_schema({
     Required("pull_request_number"): Any(All(int, Range(min=1)), None),
     Required("release_type"): text_type,
     Optional("shipping_phase"): Any('build', 'ship', None),
```

Comment:
Does `shipping_phase` not need a default?

Reply:
Apparently not? Maybe we use `config.params.get("shipping_phase")` everywhere. I guess it doesn't hurt to add one in anyway.